extensions-core/datasketches/src/main/java/org/apache/druid/query/aggregation/datasketches/theta/sql/ThetaPostAggMacros.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.query.aggregation.datasketches.theta.sql;
+
+import org.apache.druid.math.expr.Expr;
+import org.apache.druid.math.expr.ExprEval;
+import org.apache.druid.math.expr.ExprMacroTable;
+import org.apache.druid.math.expr.ExpressionType;
+import org.apache.druid.query.aggregation.datasketches.theta.SketchHolder;
+
+import javax.annotation.Nullable;
+import java.util.List;
+
+public class ThetaPostAggMacros
+{
+  public static final String THETA_SKETCH_ESTIMATE = "theta_sketch_estimate";
+
+  public static class ThetaSketchEstimateExprMacro implements ExprMacroTable.ExprMacro
+  {
+
+    @Override
+    public Expr apply(List<Expr> args)
+    {
+      validationHelperCheckArgumentCount(args, 1);
+      return new ThetaSketchEstimateExpr(args.get(0));
```

Comment:
Probably nice to add the "round" option while we are at it, makes it consistent with the HLL version.

Reply:
There are 2 options for this:
1. We only add the round to the expression and do not add it to the aggregator factory. Whatever result is returned by the aggregator factory is rounded (or not) and shown in the output.
2. We push the round as a parameter to the SketchMergeAggregatorFactory, compute it in the `finalizeComputation` method, and return the value as is.

1 is the easier way and does not affect many test cases, but 2, I believe is the better way. @imply-cheddar what do you suggest?

Reply:
After discussion we decided to leave PostAggs as is and make the expressions similar only

Reply:
Hrm, I'm okay with only implementing round in the expression, but in that case, because the expression and the PostAggregator are not consistent, you will need to stop planning the post aggregator from SQL.  I'm like 90% certain that the SQL planner is fine using an expression anywhere a post agg could do, it just "prefers" the PostAgg in certain circumstances.  So, it should be acceptable to change the SqlOperatorConversion to always return `null` from the call to build the PostAggregator and things should continue to just work.

Note, that the PostAggregator cannot be removed for compatibility reasons as it could be used from the native query APIs.

Reply:
i suppose could fall back to the expression post aggregator and use the rounding expression